data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-consensual-creation-v1.0.xml

```diff
@@ -22,7 +22,7 @@
 
         <createTable tableName="consensual_transaction_status">
             <column name="transaction_id" type="VARCHAR(160)">
-                <constraints nullable="false" foreignKeyName="fk_transaction_status" references="consensual_transaction(id)"/>
+                <constraints nullable="false" primaryKey="true" foreignKeyName="fk_transaction_status" references="consensual_transaction(id)"/>
             </column>
             <column name="status" type="VARCHAR(255)">
```

Comment:
Not related to this PR, but are not statuses 1 character long? (`U`, `V`) ?

Reply:
Yes, they are at the moment, as part of initial implementation to have it working, but status values in DB were not properly discussed, not sure whether these would be the final values.

Reply:
Ye 160 is probably too long but might not want to restrict to 1.

Do we have a tech debt to go through the schemas and make sure we're happy with everything? If not, we probably should so we can tick it off and move on at some point. Definitely a do later thing though.

Reply:
I added it here a bit earlier: https://r3-cev.atlassian.net/browse/CORE-7928